code/xcvm/lib/core/src/instruction.rs

```diff
@@ -77,6 +77,7 @@ pub enum Instruction<Payload, Account, Assets> {
 	/// The salt is used to track the program when events are dispatched in the network.
 	Spawn {
 		network: crate::network::NetworkId,
+		#[serde(serialize_with = "hex::serialize", deserialize_with = "hex::deserialize")]
 		salt: Vec<u8>,
```

Comment:
This is missing schemars `with` annotation.

Reply:
may you show example?

Reply:
```
$ git grep schemars.\*with
xcvm/lib/core/src/network.rs:   #[cfg_attr(feature = "std", schemars(with = "String"))]
xcvm/lib/core/src/network.rs:   #[cfg_attr(feature = "std", schemars(with = "String"))]
xcvm/lib/core/src/packet.rs:    #[cfg_attr(feature = "std", schemars(with = "String"))]
xcvm/lib/core/src/packet.rs:    #[cfg_attr(feature = "std", schemars(with = "String"))]
```

Reply:
thanks, seems this is no documented on schemars. used what documented.

Reply:
equivalent